_licenses/ofl-1.1.txt

```diff
@@ -1,33 +1,41 @@
 ---
 layout: license
-title: SIL Open Font License
+title: SIL Open Font License 1.1
 hidden: true
 permalink: /licenses/ofl/
+redirect_from: /licenses/ofl-1.1/
+filename: OFL.txt
 
 source: http://scripts.sil.org/OFL_web
 
-description: The Open Font License (OFL) is maintained by SIL International. It attempts to be a compromise between the values of the free software and typeface design communities. It is used for almost all open source font projects, including those by Adobe, Google and Mozilla.
+description: The Open Font License (OFL) is maintained by SIL International. It is designed to honor the values of both the Free Software and Typeface Design communities. It is used for many open font projects created by independent font developers, small foundries as well as big corporations like Adobe, Google and Mozilla.
 
-note: This license doesn't require source provision, but recommends it. All files derived from OFL files must remain licensed under the OFL.
+note: This license doesn't require source provision, but recommends it via best practices (including build scripts and related documentation). All files derived from OFL files inherit the license and must remain under the OFL.
 
-how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your font source and copy the text of the license into the file. Replace [year] with the current year and [fullname] ([email]) with the name and contact email address of each copyright holder. You may take the additional step of appending a Reserved Font Name notice. This option requires anyone making modifications to change the font's name, and is not ideal for web fonts (which all users will modify by changing formats and subsetting for their own needs.)
+how: Create a text file (typically OFL.txt) in the root of your source and copy the text of the license into the file. Fill in the header with [year], [fullname] and [email] with your information as the copyright holder (you can also add a URL to your upstream project website). As a copyright holder, you may choose to declare one - or more - Reserved Font Name(s) by adding "with Reserved Font Name &lt;Reserved Font Name&gt;" under your copyright statement. This is an optional feature which requires other developers who make modifications to change the font name and pick their own unique name to prevent collisions and respect your canonical branch. When developing an open font collaboratively in a public repository, you can also use a development name that is different from the release name which users expect to be stable. Documenting changes in a FONTLOG.txt file is recommended. See the <a href="http://scripts.sil.org/OFL-FAQ_web">OFL-FAQ</a> for more details and practical tips.
```

Comment:
for 2. This has been the recommended standard best practise for the community using the license. Even before github appeared :-)  Why would it be such a problem that there is a specific filename that font authors more easily recognize? My sense is that too few people look at the content of a generically-named license file and can't tell what licensing it actually is available under. Being explicit in the filename itself is a conscious decision to reserve tendencies to conveniently sweep that under the carpet. Mandating other filenames would fragment the existing community. (BTW other hosting services are more explicit in showing which actual license is picked on a project page).